adobe/react-spectrum
# fix(#2621): ActionGroup with overflowMode: collapse: ActionMenu button should not receive aria-checked

Closes #2621

## ✅ Pull Request Checklist:

- [x] Included link to corresponding [React Spectrum GitHub Issue 2621](https://github.com/adobe/react-spectrum/issues/2621).

## 📝 Test Instructions:

1. Open https://reactspectrum.blob.core.windows.net/reactspectrum/4b281c9b7f1298120d23bb59c1453c7e091c433d/storybook/index.html?path=/story/actiongroup--overflowmode-collapse-selection
2. Resize container so that items in ActionGroup collapse into menus.
3. Verify that after rerunning tests in Storybook Accessibility panel, there are no issues reported for `aria-checked` used inappropriately on a `button` element.

## 🧢 Your Project:

Adobe/Accessibility
